tools/automation/cli_linter/rules/exclusions.json

```diff
@@ -0,0 +1,24 @@
+{
+    "help_entries": {
+        "example_entry": [
+            "unrecognized_help_entry_rule"
```

Comment:
I think we should definitely reconsider this. A single dumping ground for all exceptions would, I think, grow unwieldy. I think, as Derek suggested, having an exception file per module/extension is the better way to go.

Reply:
Hmm, that wouldn't be applicable to the help_entry rules and it seems like a lot of extra files that may only have one rule exception in them. It would be the same information just scattered throughout the modules.

Having one source gives the benefit of making it very easy to quickly search for and add new exclusions. Maybe we can save this discussion when we have more exclusions and see which is more usable?

Reply:
This piece should be vetted through the whole team, but I see some of your points.

I would also add that JSON is a pretty unfriendly format and would recommend something else (YAML, config format, etc.)